src/scss/components/_login-button-logo.scss

```diff
@@ -0,0 +1,20 @@
+@use 'sass:math';
+
+@import '~microscope-sass/lib/typography';
```

Comment:
also unused, and I think it should be `responsive` instead of `typography` for the `mobile-only` mixin, but it kinda works because another file already loaded it.

We should refactor into using proper sass modules at some point :grimacing:

Reply:
I thought that typography was needed for the `$grid-row-height`, but that's in `grid` and `typography` imports `grid`. So I removed `typography`, added `grid` and `responsive`